contracts/base/LoansBase.sol

```diff
@@ -582,6 +595,15 @@ contract LoansBase is LoansInterface, Base {
         @param amount The amount of collateral to be paid
      */
     function _payInCollateral(uint256 loanID, uint256 amount) internal {
+        if (collateralToken == _getSettings().ETH_ADDRESS()) {
+            // Ether collateral
+            require(msg.value == amount, "INCORRECT_ETH_AMOUNT");
```

Comment:
we should change the collateral token to be the `WETH` contract address that is on settings

also here is the point in which we should wrap the ETH value in WETH. after this point the only other point the contract needs to know if the collateral token is ETH or not is upon withdrawing. then we should be unwrapping the WETH and sending the caller back their unwrapped ETH collateral.

Reply:
Done